nvidia_triton/datadog_checks/nvidia_triton/check.py

```diff
@@ -0,0 +1,111 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from urllib.parse import urljoin  # noqa: F401
+
+from six.moves.urllib.parse import urlparse
+
+from datadog_checks.base import AgentCheck, OpenMetricsBaseCheckV2
+from datadog_checks.base.errors import ConfigurationError
+
+from .metrics import METRICS_MAP
+
+DEFAULT_METADATA_ENDPOINT = '/v2'
+DEFAULT_HEALTH_ENDPOINT = DEFAULT_METADATA_ENDPOINT + '/health/ready'
+
+
+class NvidiaTritonCheck(OpenMetricsBaseCheckV2):
+
+    # This will be the prefix of every metric and service check the integration sends
+    DEFAULT_METRIC_LIMIT = 0
+    __NAMESPACE__ = 'nvidia_triton'
+
+    def __init__(self, name, init_config, instances=None):
+        super(NvidiaTritonCheck, self).__init__(name, init_config, instances)
+        if 'openmetrics_endpoint' not in self.instance:
```

Comment:
Hmm, do you have a test that confirms we need this check? I thought that if the field was required, we would crash earlier than this.